src/app/@ansyn/overlays/actions/overlays.actions.ts

```diff
@@ -173,6 +176,19 @@ export class SetOverlaysStatusMessage implements Action {
 	}
 }
 
+export class SetHoveredOverlay implements Action {
+	type = OverlaysActionTypes.SET_HOVERED_OVERLAY;
+
+	constructor(public payload: HoveredOverlayData) {
+	}
+}
+
+export class ClearHoveredOverlay implements Action {
+	type = OverlaysActionTypes.CLEAR_HOVERED_OVERLAY;
+
+	constructor(public payload?: string) {
```

Comment:
remove `constructor`

Reply:
If I remove the constructor, I get error messages in ng serve, from the reducer:

RROR in src/app/@ansyn/overlays/reducers/overlays.reducer.ts(104,34): error TS2339: Property 'payload' does not exist on type 'OverlaysActions'.
  Property 'payload' does not exist on type 'ClearHoveredOverlay'.
src/app/@ansyn/overlays/reducers/overlays.reducer.ts(105,26): error TS2339: Property 'payload' does not exist on type 'OverlaysActions'.
  Property 'payload' does not exist on type 'ClearHoveredOverlay'.
src/app/@ansyn/overlays/reducers/overlays.reducer.ts(113,43): error TS2339: Property 'payload' does not exist on type 'OverlaysActions'.
  Property 'payload' does not exist on type 'ClearHoveredOverlay'.......

Reply:
So, can I keep the constructor? :)